tests/cupy_tests/core_tests/test_ndarray_indexing.py

```diff
@@ -56,8 +52,6 @@
     {'shape': (10,), 'transpose': None, 'indexes': (slice(-5, -10, -1),)},
     {'shape': (10,), 'transpose': None, 'indexes': (slice(-5, -11, -1),)},
     {'shape': (10,), 'transpose': None, 'indexes': (slice(-5, -12, -1),)},
-    {'shape': (10,), 'transpose': None,
```

Comment:
Could you move restore this test case and move it to `test_ndarray_adv_indexing.py` as well?